testing/tests/DevExpress.ui.widgets/tabPanel.width.tests.js

```diff
@@ -15,6 +16,8 @@ QUnit.module('Tabs width', () => {
             this.$tabPanel = $('<div>');
             this.assert = assert;
             this.setWidthApproach = setWidthApproach;
+
+            addShadowDomStyles($('#qunit-fixture'));
```

Comment:
why should we do this? I thought that shadow dom styles are added on component render

Reply:
I'll check it. Probably it's a fix that is not actual anymore

Reply:
not needed, thank you for pointing out